src/gui/dialogs/addon/manager.cpp

```diff
@@ -836,15 +836,19 @@ void addon_manager::execute_action_on_selected_addon()
 		find_widget<button>("details_toggle").set_label(_("Add-on Details"));
 	}
 
+	// The pointer returned by get_selected_addon comes from the vector<addon_info*> owned by the addon_list.
+	// This is a pointer into the addons_list map owned by the addon_manager.  Any modification to the map
+	// kicked off by this method call will cause that pointer to dangle.  Copying the addon_info here prevents
+	// that.  
```

Comment:
Just noting there's trailing white-space here that's triggering a CI error.

Reply:
I think it would be clearer to put this comment above the line that makes the copy.